src/main/java/it/eng/idsa/businesslogic/service/impl/HttpHeaderServiceImpl.java

```diff
@@ -94,7 +92,7 @@ public Map<String, Object> messageToHeaders(Message message) {
 			}
 		});
 
-		logger.debug("Message converted, following headers are the result: \r\n {}", headers.toString());
+		logger.debug("Message converted");
```

Comment:
Maybe add in that if(debugEnabled) to print what is the result message, from that Helper method?
Unless we have the result of this method logged somewhere
Same applies for other method in this class

Reply:
I will add it